ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/gio-formly-json-schema.service.ts

```diff
@@ -28,6 +28,10 @@ export class GioFormlyJsonSchemaService {
   public toFormlyFieldConfig(jsonSchema: GioJsonSchema): FormlyFieldConfig {
     return this.formlyJsonschema.toFieldConfig(jsonSchema, {
       map: (mappedField: FormlyFieldConfig, mapSource: JSONSchema7) => {
+        if (mapSource.deprecated) {
```

Comment:
It's possible to encapsulate this like the others in a function : 
```
        mappedField = this.deprecatedMap(mappedField, mapSource);
```
🥺

Reply:
yes sure